crowbar_framework/spec/lib/crowbar/repositories_spec.rb

```diff
@@ -0,0 +1,84 @@
+require "spec_helper"
+
+describe Crowbar::Repository do
+  before do
+    Proposal.create!(barclamp: "provisioner", name: "test")
+  end
+  let!(:registry) { Crowbar::Repository.require_registry }
```

Comment:
This is what I meant by [my other comment](https://github.com/crowbar/crowbar-core/pull/18#discussion_r38771085).  If I didn't know the code already, I would expect `require_registry` to return `true` or `false` depending on whether it successfully loaded the registry, just like `Kernel#require` and `Kernel#require_relative` return a Boolean.